src/main/java/io/confluent/idesidecar/restapi/credentials/KerberosCredentials.java

```diff
@@ -2,6 +2,7 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.confluent.idesidecar.restapi.exceptions.Failure.Error;
+import io.confluent.idesidecar.restapi.util.OperatingSystemType;
```

Comment:
unused?

Reply:
Good catch! I forgot to clean it up. Removed in [53f3566](https://github.com/confluentinc/ide-sidecar/pull/422/commits/53f356611847ebd116de271689e2780536ddd83a).